### PR TITLE
Page branding

### PR DESCRIPTION
**Pull request by MelanyCristy**
Me and Pinelopi changed most of the style. Still a little things to on work on but so far we can say it's done the first step.

### CODE REVIEW

File: frontend/package-lock.json [hunk 1]
```diff
@@ -9,28 +9,28 @@
...
-        "@babel/traverse": "7.6.3",
-        "@babel/types": "7.6.3",
-        "convert-source-map": "1.6.0",
-        "debug": "4.1.1",
-        "json5": "2.1.1",
-        "lodash": "4.17.15",
-        "resolve": "1.12.0",
-        "semver": "5.7.1",
-        "source-map": "0.5.7"
+        "@babel/code-frame": "^7.5.5",
```
Comment (SaraGarabedian) [inline frontend/package-lock.json:20]: What does this change do?
Why are the versions changed?

**Review by krismutalapova — APPROVED** [review]
LGTM

File: frontend/src/components/home/HomePage.js [hunk 2]
```diff
@@ -5,14 +5,14 @@ class HomePage extends Component {
...
-                        <li><a href="https://spring.io/projects/spring-boot">Spring</a></li>
-                        <li><a href="https://www.postgresql.org">PostgreSQL</a></li>
-                        <li><a href="https://reactjs.org">React</a></li>
-                        <li><a href="https://reacttraining.com/react-router/web/guides/quick-start">React Router</a></li>
-                        <li><a href="https://github.com/axios/axios">Axios</a></li>
+                        <li><a href=""></a></li>
+                        <li><a href=""></a></li>
+                        <li><a href=""></a></li>
+                        <li><a href=""></a></li>
+                        <li><a href=""></a></li>
```
Comment (SaraGarabedian) [inline frontend/src/components/home/HomePage.js]: Why keep the tags for the ul when emptied?

File: frontend/src/index.css [hunk 3]
```diff
@@ -1,10 +1,11 @@
 body {
-  margin: 0;
+   margin: 0;
```
Comment (SaraGarabedian) [inline frontend/src/index.css]: Unnecessary formatting, should be removed

**Review by SaraGarabedian — COMMENTED** [review]
Could you rephrase your commit, so it says what it does, such as:
Instead of "Me and Pinelopi changed most of the style. Still a little things to on work on but so far we can say it's done the first step."
Something like:
"Added a new logo, changed background, updated versions, etc."

**Comment by MelanyCristy** [comment]
> Could you rephrase your commit, so it says what it does, such as:
> Instead of "Me and Pinelopi changed most of the style. Still a little things to on work on but so far we can say it's done the first step."
> Something like:
> "Added a new logo, changed background, updated versions, etc."

**This is the other way around. We did as you mentioned, the first message is just a pull request description. The commit messages are the ones with the description of our commits**
